src/modules/swap/ModuleSwap.vue

```diff
@@ -782,6 +782,9 @@ export default {
     },
     mainTokenDetails() {
       this.setDefaults();
+    },
+    amountErrorMessage(newVal) {
+      if (newVal !== '') this.availableQuotes.splice(0);
```

Comment:
Will it always be the 0th value to splice?

Reply:
yeah, it's resetting the array back to empty